tests/pdifftests/test.py

```diff
@@ -26,15 +27,18 @@ class Runs(Base):
     window_size = [1280, 800]
     base_url = BASE
 
+    @hide('span.time')
```

Comment:
Yeah, hiding those would be good; I noted in the commit msg that I hide those which were easily selectable. Didn't want to do nth-child selections to hide content; too brittle. Need to add a css class to the span containing the time in examine_comments-XX; I was thinking it could be done in another PR.